bpf/mock/Makefile

```diff
@@ -22,13 +22,13 @@ check_helper_headers: generate_helper_headers mock_helpers
 	# Generate temp headers from lib/helpers.h, lib/helpers_skb.h and lib/helpers_xdp.h.
 	# Compare them to the headers within the current directory.
 	# The temp headers are removed after finishing.
-	rm -r mocks;
+	$(QUIET)rm -r mocks;
 
 generate_helper_headers:
 	# Auto-generate helpers.h, helpers_skb.h, and helpers_xdp.h.
 	# Take contents from bpf/helpers.h, bpf/helpers_skb.h and bpf/helpers_xdp.h.
 	# Prune to be ready to mock and save to the current directory.
-	mkdir -p mocks; \
+	$(QUIET)mkdir -p mocks; \
```

Comment:
Nit: Would be nice to indent the `sed` lines below, I didn't understand at first that their were the continuation of the same command and was wondering why they didn't get their `$(QUIET)`.

Reply:
I'm actually not sure why there's all these `;` throughout this Makefile, probably the original author didn't fully understand the Makefile syntax and how this causes the rest of the commands to be shelled out and to ignore the result of each command. Would be a nice cleanup but yeah seems like we could follow up separately on this.

Reply:
Right, didn't pay attention to that but now that you mention it, they should be separate commands entirely.
Agreed that this is for a future PR :+1:.

Reply:
^^ I've added a new commit on top to resolve this as well, PTAL.